mybatis-spring-boot-autoconfigure/src/test/java/org/mybatis/spring/boot/autoconfigure/MybatisAutoConfigurationTest.java

```diff
@@ -164,6 +166,27 @@ public void testWithDatabaseIdProvider() {
 		assertEquals("h2", this.context.getBean(SqlSessionFactory.class).getConfiguration().getDatabaseId());
 	}
 
+	@Test
+	public void testMyBatisConfigurationUsingConfigurationProperties() {
```

Comment:
Add minimum assertion on `testDefaultConfiguration()` via 5670187f7b483125be4f06e145cf806ef36143e0.
Should i added a new test case ?